content/docs/reactivesearch/vue/overview/QuickStart.md

```diff
@@ -131,11 +144,36 @@ We will update our `src/App.vue` file to add the ReactiveBase component.
 
 This is how the app should look after running the `yarn run serve` command.
 
-![Image](https://i.imgur.com/6gKpAEI.png)
+<iframe src="https://codesandbox.io/embed/summer-currying-l0mf0?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+	style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+	title="summer-currying-l0mf0"
+	allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+	sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+></iframe>
 
 ---
 
-### Step 4: Adding Filters and Result Components
+### Step 4: Adding Search Component
+
+For this we will be using [data-search](/docs/reactivesearch/vue/search/DataSearch/) component that creates a search box UI component that is connected to one or more database fields. In this scenario, search can be implemeted on title of the book or on the author of the book.
+
+```html
+<data-search
+	componentId="SearchBox"
+	className="search-container"
+	:dataField="[
+		'authors',
+		'authors.autosuggest',
+		'original_title',
+		'original_title.autosuggest',
+	]"
+	:fieldWeights="[3, 1, 5, 1]"
+/>
+```
+
+We need to add this within the ReactiveBase component.
+
+### Step 5: Adding Filters and Result Components
```

Comment:
@lohi212 We are missing the explanation and snippets for the filter components. You can follow the React quickstart guide here.

Reply:
I would suggest using the same structure.